src/main/java/test/project/bookshop/repository/book/BookRepository.java

```diff
@@ -1,12 +1,20 @@
 package test.project.bookshop.repository.book;
 
+import java.util.Optional;
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.EntityGraph;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
 import test.project.bookshop.model.Book;
 
 public interface BookRepository extends JpaRepository<Book, Long>,
         JpaSpecificationExecutor<Book> {
     Page<Book> findAllByCategoriesId(Long categoryId, Pageable pageable);
+
+    @EntityGraph(attributePaths = {"categories"})
+    Page<Book> findAll(Pageable pageable);
```

Comment:
Let's be more obvious and informative
```suggestion
    Page<Book> findAllFetchCategories(Pageable pageable);
```